arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/nview/NListView.java

```diff
@@ -12,13 +15,24 @@
  */
 public class NListView extends NNodeView implements ListRootDisplay
 {
+    private static final Logger LOG = Logger.getLogger(NListView.class.getName());
+
     private final MaterialButton addButton = new MaterialButton();
 
+    // private final FlowPanel inputGroupPanel = new FlowPanel();
+
+    public NListView()
+    {
+        // LOG.severe(">> asking to create a listView");
```

Comment:
We probably do want to display a pretty explicit error message here, at least as long as the class is not implemented.